docs/guides/host.md

```diff
@@ -59,11 +65,11 @@ prefect config set PREFECT_API_URL="http://127.0.0.1:4200/api"
 !!! tip "`PREFECT_API_URL` required when running Prefect inside a container"
     You must set the API server address to use Prefect within a container, such as a Docker container.
 
-    You can save the API server address in a [Prefect profile](/concepts/settings/). Whenever that profile is active, the API endpoint will be be at that address.
+    You can save the API server address in a [Prefect profile](/concepts/settings/). Whenever that profile is active, the API endpoint is at that address.
 
     See [Profiles & Configuration](/concepts/settings/) for more information on profiles and configurable Prefect settings.
 
-## Prefect database
+## About Prefect database
```

Comment:
```suggestion
## The Prefect database
```

Reply:
"About Prefect database" sounds strange, but I'm not sure I love "The Prefect database" either.